src/validation/date.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
       mappings: this.valueOrRefOptions
     });
 
+    if(typeDescriptor.nullable) {
```

Comment:
Can we move this logic to this file https://github.com/talyssonoc/structure/blob/master/src/validation/utils.js#L55? It would remove some of the duplication.